sqlglot/expressions.py

```diff
@@ -6414,6 +6414,10 @@ class ToBase64(Func):
     pass
 
 
+class Base64DecodeBinary(Func):
+    pass
```

Comment:
This isn't right, we need to set `arg_types = {"this": True, "alphabet": False}`, as the function supports two arguments:

```
BASE64_DECODE_BINARY( <input> [ , <alphabet> ] )
```

Let's make sure we have a `validate_identity` test that covers both arguments.